lib/gravity.rb

```diff
@@ -0,0 +1,28 @@
+# API Client for Gravity V1
+class Gravity
+  GRAVITY_V1_API_URL = Rails.application.config_for(:gravity)["api_v1_root"]
+
+  class GravityError < StandardError; end
+  class GravityNotFoundError < GravityError; end
+
+  def self.get(url:, additional_headers: {}, params: {})
+    response = Faraday.get(url, params, headers.merge(additional_headers))
```

Comment:
imho Faraday is just plain nicer to work with. It's more simple at the call site and easier to work with when it comes to testing. The default adapter is net/http so that's ultimately what's being used but yeah the interface to working with it is just so much lamer. 🤷

Reply:
Yes! A bit of a personal preference with Faraday
Also more of our "high usage" ruby apps like Exchange/Gravity are already using it too!

There is a bit of an opportunity for consolidating these Gravity clients / HTTP libraries across Artsy as well
https://artsy.slack.com/archives/C05F8TNKGAV/p1726152067455139